### PR TITLE
Update qtwebkit

### PR DESCRIPTION
**Pull request by tehnick**

### CODE REVIEW

File: src/qtwebkit.mk [hunk 1]
```diff
@@ -4,11 +4,11 @@ PKG             := qtwebkit
...
 $(PKG)_DESCR    := QtWebKit
 $(PKG)_IGNORE   :=
-$(PKG)_VERSION  := 5.212
-$(PKG)_CHECKSUM := fada6fa2f2bae27a16b666e0360680de6ab21aac044670b434a729334e57b154
+$(PKG)_VERSION  := 5.13.1
+$(PKG)_CHECKSUM := de062c94dcc163b935f4f957a08f5b32cfcb5f15cdfb67ce8a036ce4f222edb3
 $(PKG)_SUBDIR   := qtwebkit-everywhere-src-$($(PKG)_VERSION)
 $(PKG)_FILE     := qtwebkit-everywhere-src-$($(PKG)_VERSION).tar.xz
-$(PKG)_URL      := https://download.qt.io/snapshots/ci/qtwebkit/$($(PKG)_VERSION)/latest/src/submodules/$($(PKG)_FILE)
+$(PKG)_URL      := https://download.qt.io/snapshots/ci/qtwebkit/5.212/latest/src/submodules/$($(PKG)_FILE)
```
Comment (tonytheodore) [inline src/qtwebkit.mk]: Why would the file not be in the versioned path?

Comment (tehnick) [reply src/qtwebkit.mk]: Ask Qt developers why they have put QtWebkit 5.13.1 into the path containing 5.212...

**Comment by tehnick** [comment]
All was built sucessfully for targets:
```
MXE_TARGETS := i686-w64-mingw32.shared x86_64-w64-mingw32.shared
```
And works without noticeable change at the first glance.

**Comment by tonytheodore** [comment]
Thanks!

**Comment by tonytheodore** [comment]
@tehnick the checksum appears to have changed - are you able to compare your original file with the one that's available now?

**Comment by tehnick** [comment]
Yes, they did something strange:
```
$ LANG=C diff -urN qtwebkit-everywhere-src-5.13.1.tar.xz__2019-10-11 qtwebkit-everywhere-src-5.13.1.tar.xz__2019-10-14
Binary files qtwebkit-everywhere-src-5.13.1.tar.xz__2019-10-11 and qtwebkit-everywhere-src-5.13.1.tar.xz__2019-10-14 differ

$ LANG=C diff -urN qtwebkit-everywhere-src-5.13.1__2019-10-11/ qtwebkit-everywhere-src-5.13.1__2019-10-14/ | wc -l
0
```
Probably they just re-generated tarball with new timestamps.

**Comment by tonytheodore** [comment]
Thanks, fixed in https://github.com/mxe/mxe/pull/2414#issuecomment-542245501

**Pull request merged**